### ISSUE TITLE
Login/register with firebase 

### ISSUE THREAD

**Issue by tiago-carvalho-deltatre** [razaghimahdi/Shopping-By-KMP#4]
Are you planing on implementing google/apple sign in with kmm?

**Comment by razaghimahdi** [comment]
Yeah, actually one of our future plans is to implementing google/apple sign-in. it will be add soon.

**Comment by simplekjl** [comment]
@razaghimahdi can I help with that development?

**Comment by razaghimahdi** [comment]
> @razaghimahdi can I help with that development?

I would love to, but its apis aren't ready yet, I'm not sure how long gonna take to get the api, also backend is open-source so if you have knowledge, we would love that u make contribute with that either

**Comment by simplekjl** [comment]
I can do that, maybe we can host all together on Firebase

**Comment by razaghimahdi** [comment]
Sure, but u need to add api key to the repository, since it is a sample project on Jetbrains. So both google & apple right?

**Comment by SamuelMarks** [comment]
Related: https://github.com/mirzemehdi/KMPAuth
